gen_huishoudelijkafvalservicegebieden.py

```diff
@@ -0,0 +1,155 @@
+import re
+import random
+
+from generate import block, header, p, q
+
+
+def slugify(s: str) -> str:
+    # TODO would be cleaner to convert to NFD, then remove combining chars.
+    s = s.replace("ë", "e")
+    return re.sub(r"[^A-Za-z]+", "_", s).strip("_").lower()
+
+layers = [
+    ("Servicegebieden", "Servicegebieden Rest", "Rest"),
+    ("Servicegebieden", "Servicegebieden Glas", "Glas"),
+    ("Servicegebieden", "Servicegebieden Papier", "Papier"),
+    ("Servicegebieden", "Servicegebieden Textiel", "Textiel"),
+    ("Servicegebieden", "Servicegebieden GFT", "GFT"),
+    ("Servicegebiedenlocatie", "Servicegebiedenlocatie Rest", "Rest"),
+    ("Servicegebiedenlocatie", "Servicegebiedenlocatie Glas", "Glas"),
+    ("Servicegebiedenlocatie", "Servicegebiedenlocatie Papier", "Papier"),
+    ("Servicegebiedenlocatie", "Servicegebiedenlocatie Textiel", "Textiel"),
+    ("Servicegebiedenlocatie", "Servicegebiedenlocatie GFT", "GFT")
+]
+
+colors = ("#8dd3c7", "#ffffb3", "#bebada", "#fb8072", "#80b1d3", "#fdb462", "#b3de69", "#fccde5", "#d9d9d9", "#bc80bd")
+
+
+with block("MAP"):
+    p("NAME", "huishoudelijkafvalservicegebieden")
+    p("INCLUDE", "header.inc")
+
+
+    with block("WEB"):
+        with block("METADATA"):
+            q("ows_title", "Huishoudelijkafval Servicegebieden")
+            q("ows_onlineresource", "MAP_URL_REPLACE/maps/huishoudelijkafvalservicegebieden")
+            q("ows_abstract", "Servicegebieden van huishoudelijkafvalcontainers",)
+
+    
+    for group, layer_name, filter_value in layers:
+
+        if group == 'Servicegebieden':
+
+            with block("LAYER"):
+                sql = f"geometrie from (SELECT *, (regexp_matches(servicegebieden_locatie_id, '[0-9](?!.*[0-9])'))[1] AS last_numeric_digit FROM huishoudelijkafval_servicegebieden where fractie_omschrijving = '{filter_value}') as subquery USING srid=28992 USING UNIQUE id"
+                
+
+                layer_name_slug = slugify(layer_name)
+
+                p("NAME", layer_name_slug)
+                p("INCLUDE", "connection/dataservices.inc")
+                p("DATA", sql)
+                p('GROUP', slugify(group))
+                p("TYPE POLYGON")
+                p("TEMPLATE", "fooOnlyForWMSGetFeatureInfo.html")
+
+                with block("PROJECTION"):
+                    q("init=epsg:28992")
+                
+                with block("METADATA"):
+                    q("ows_title", layer_name)
+                    q('ows_group_title', group)
+                    q("wms_include_items", "all")
+                    q("wms_enable_request", "*")
+                    q("ows_abstract", "Servicegebieden")
+                    q("wms_srs", "EPSG:28992")
```

Comment:
Already set in [`header.inc` line 51](https://github.com/Amsterdam/mapserver/blob/39a5a693206aeb841b37954cc62a474a1fa6a198/header.inc#L51)